src/pt_br/index.js

```diff
@@ -582,6 +634,18 @@ export const i19ChannelType = {
   chatbot: 'Chatbot'
 }
 
+export const i19CommodityType = {
+  physical: 'Físico',
+  digital: 'Digital'
+}
+
+export const i19Condition = {
```

Comment:
```suggestion
export const i19ProductCondition = {
```

Melhor um nome mais descritivo aqui, não?